src/calculateSimilarity.ts

```diff
@@ -0,0 +1,39 @@
+import { disassembleHangul } from './disassemble';
+
+export function calculateSimilarity(a: string, b: string): number {
+  const disassembleAndClean = (text: string) => disassembleHangul(text).replace(/\s/g, '');
+
+  const disassembledA = disassembleAndClean(a);
+  const disassembledB = disassembleAndClean(b);
+
+  const maxLength = Math.max(disassembledA.length, disassembledB.length);
+  if (maxLength === 0) {
+    return 100;
+  }
+
+  const distance = levenshtein(disassembledA, disassembledB);
+  const similarity = ((maxLength - distance) / maxLength) * 100;
```

Comment:
```suggestion
  const similarity = ((maxLength - distance) / maxLength);
```

유사도를 표현할 때 퍼센테이지를 표현하고 싶다면 100%가 1이라면 어떨까요? 인터페이스를 사용하는 입장에서 매번 100을 나눠야하는 경우가 많을 거 같아서요

```ts
const result = width * calculateSimilarity(...) / 100 // 이런 패턴이 자주 보일 거 같아요
```

제안코드대로면 / 100이 없을 거 같아요

Reply:
너무 좋은 의견이라고 생각합니다! 
0~100 을 제공하기 보다는 0.00 ~ 1.00 형태로 제공해주도록 수정할게요! 
이유는 말씀해주신 내용 + 다음과 같습니다!

>표준화된 표현: 0.00 ~ 1.00 범위는 많은 라이브러리와 알고리즘에서 유사도를 나타내기 위해 사용하는 표준화된 표현입니다.
정밀도: 소수점을 포함한 범위는 미세한 차이를 더 명확하게 나타낼 수 있습니다.
범용성: 0.00 ~ 1.00 사이의 값은 확률, 비율 등의 다른 측정과 쉽게 통합될 수 있습니다.

Reply:
[유사도를 0 ~ 100 이 아닌, 0 ~ 1 수로 나타내기:](https://github.com/toss/es-hangul/pull/104/commits/2b2166e0d69dc821df0c8fa6e532ec234d0f56f0)

Reply:
해당 코멘트 반영된 것 확인하여 Resolve 하겠습니다!
cc. @manudeli